crates/goose/src/providers/ollama.rs

```diff
@@ -273,3 +317,46 @@ impl OllamaProvider {
         filtered
     }
 }
+
+#[async_trait]
+impl EmbeddingCapable for OllamaProvider {
+    async fn create_embeddings(&self, texts: Vec<String>) -> Result<Vec<Vec<f32>>> {
```

Comment:
Why are we adding these implementations of `create_embeddings`? It doesn't seem to be relevant to the rest of the change. I could be missing something...